documentation/asciidoc/computers/configuration/raspi-config.adoc

```diff
@@ -246,7 +239,7 @@ Display a description of `raspi-config`.
 [[finish]]
 === Finish
 
-Exits `raspi-config`. If you made changes that require a reboot, `raspi-config` prompts you to reboot. When implementing changes for the first time, it's best to reboot. If you chose to resize your SD card, rebooting may take longer than usual.
+Exit `raspi-config`. If necessary, raspi-config will ask you to reboot when you exit. When implementing changes for the first time, reboot to ensure your changes take effect.
```

Comment:
```suggestion
Exit `raspi-config`. If necessary, `raspi-config` will ask you to reboot when you exit. When implementing changes for the first time, reboot to ensure your changes take effect.
```
Suggestion: consistently refer to raspi-config in monospace text, since it's both a program and a command.